docs/user/03-traces.md

```diff
@@ -411,7 +411,7 @@ By default, all engines for the [Serverless](https://kyma-project.io/#/serverles
 
 ## Operations
 
-A TracePipeline creates a Deployment running OTel Collector instances in your cluster. That instances will serve OTLP endpoints and ship received data to the configured backend. The Telemetry module assures that the OTel Collector instances are operational and healthy at any time. The Telemetry module delivers the data to the backend using typical patterns like buffering and retries (see [Limitations](#limitations)). However, there are scenarios where the instances will drop logs because the backend is either not reachable for some duration, or cannot handle the log load and is causing back pressure.
+A TracePipeline creates a Deployment running OTel Collector instances in your cluster. That instance serves OTLP endpoints and ships received data to the configured backend. The Telemetry module assures that the OTel Collector instances are operational and healthy at any time. The Telemetry module delivers the data to the backend using typical patterns like buffering and retries (see [Limitations](#limitations)). However, there are scenarios where the instances will drop traces because the backend is either not reachable for some duration, or cannot handle the traces load and is causing back pressure.
```

Comment:
```suggestion
A TracePipeline creates a Deployment running OTel Collector instances in your cluster. That Deployment serves OTLP endpoints and ships received data to the configured backend. The Telemetry module assures that the OTel Collector instances are operational and healthy at any time. The Telemetry module delivers the data to the backend using typical patterns like buffering and retries (see [Limitations](#limitations)). However, there are scenarios where the instances will drop traces because the backend is either not reachable for some duration, or cannot handle the traces load and is causing back pressure.
```